templates/daily-word/README.md

```diff
@@ -0,0 +1,19 @@
+# Daily Word project template
+
+This is an example of how to use the Playpass SDK to build a simple daily word game.
+
+Preview: https://daily-word--template.playpass.games
```

Comment:
@aduros Is CI set up to create this when the template is updated?

Reply:
Yup, on commits to `main` all templates are deployed to `https://${TEMPLATE_NAME}--template.playpass.games`